code/go/0chain.net/blobbercore/handler/grpc_helpers.go

```diff
@@ -18,18 +19,18 @@ type gRPCHeaderMetadata struct {
 	ClientSignature string
 }
 
-
 func registerGRPCServices(r *mux.Router, server *grpc.Server) {
 	blobberService := newGRPCBlobberService()
 	r.Use(Middleware2("ds"))
 	grpcGatewayHandler := runtime.NewServeMux(
-								runtime.WithIncomingHeaderMatcher(CustomMatcher),
-							)
-
+		runtime.WithIncomingHeaderMatcher(CustomMatcher),
+	)
 
 	blobbergrpc.RegisterBlobberServiceServer(server, blobberService)
 	_ = blobbergrpc.RegisterBlobberServiceHandlerServer(context.Background(), grpcGatewayHandler, blobberService)
 	r.PathPrefix("/").Handler(grpcGatewayHandler)
+
+	grpcHandlePaths(grpcGatewayHandler)
```

Comment:
Cant we do directly do this ? Why to add an extra function hope in this
```suggestion


	grpcGatewayHandler.HandlePath("POST", "/v1/file/upload/{allocation}", common.UserRateLimit(common.ToJSONResponse(WithConnection(UploadHandler)))
```

Reply:
Fixed

Reply:
awesome!